### DISCUSSION TITLE
Automation of administrator actions

### DISCUSSION THREAD

**Discussion by jussara-ti** [nilsteampassnet/TeamPass#3624] [Q&A] [answered]
Hello, 

I'm wondering about the actions in the Teampass administrator view. The ideal, for me, is that these actions were automatic, principally the action "create the personal folder for all users without one". Or, at least for this one, the manager should have the permission to execute this, because if he have the right to create a user, he should have the right to execute this action too. It's impractical to the admin login only to start this action. 

Currently, there's some way to do these actions automatic? If not, can I do this through the API?

**Comment by nilsteampassnet** [comment]
Interresting ... let me think about it

**Comment by nilsteampassnet** [comment, marked as answer]
@jussara-ti I had implemented it in 3.0.9
